config-lib/src/main/java/com/yahoo/config/FileNode.java

```diff
@@ -14,6 +16,8 @@ public FileNode() {
     public FileNode(String stringVal) {
         super(true);
         this.value = new FileReference(ReferenceNode.stripQuotes(stringVal));
+        if (Path.of(value.value()).normalize().startsWith(".."))
```

Comment:
Why not use vespajlib Path and you get this for free?

Reply:
This module didn't already depend on that. I guess I could have changed instead, but I opted for this.